src/platform/Linux/PlatformManagerImpl.cpp

```diff
@@ -187,6 +276,48 @@ CHIP_ERROR PlatformManagerImpl::_GetCurrentHeapHighWatermark(uint64_t & currentH
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR PlatformManagerImpl::_GetRebootCount(uint16_t & rebootCount)
+{
+    uint32_t count = 0;
+
+    CHIP_ERROR err = ConfigurationMgrImpl().GetRebootCount(count);
+
+    if (err == CHIP_NO_ERROR)
+        rebootCount = count;
```

Comment:
As below, this would not compile with `-Wconversion` and we should make sure our code does compile like that.